### PR TITLE
Release 10.0.0

### PR DESCRIPTION
**Pull request by martinsrenato**
PW-4417 Pass brand value for giftcard #245 
PW-4417 Added current version information in README.md #246
PW-4462 Updated to checkout v67 #249
PW-1905 - Added support to Googlepay #250

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 This plugin supports SAP Commerce (Hybris) versions 6.x
 
+The plugin is using following adyen libraries and API.
+- [adyen-java-api-library](https://github.com/Adyen/adyen-java-api-library) (v12.0.1)
```
Comment (Aleffio) [inline README.md:6]:
```suggestion
- [adyen-java-api-library](https://github.com/Adyen/adyen-java-api-library) (v14.0.0)
```

File: README.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
...
+- [adyen-web](https://github.com/Adyen/adyen-web) (v3.23.0)
+- [Adyen Checkout API](https://docs.adyen.com/api-explorer/) (v66)
```
Comment (Aleffio) [inline README.md:8]:
```suggestion
- [Adyen Checkout API](https://docs.adyen.com/api-explorer/) (v67)
```

**Review by Aleffio — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -2,6 +2,11 @@
 
 This plugin supports SAP Commerce (Hybris) versions 6.x
 
+The plugin is using following adyen libraries and API.
+- [adyen-java-api-library](https://github.com/Adyen/adyen-java-api-library) (v14.0.0)
+- [adyen-web](https://github.com/Adyen/adyen-web) (v3.23.0)
```
Comment (saquibsayyad) [inline README.md:7]: components are also updated to `4.3.1`

Comment (martinsrenato) [reply README.md]: Fixed

**Review by Aleffio — APPROVED** [review]

**Review by saquibsayyad — APPROVED** [review]

**Review by rkewlani — APPROVED** [review]

**Pull request merged**